### ISSUE TITLE
override value matched by handler?

### ISSUE THREAD

**Issue by FezVrasta** [clement-escolano/parse-torrent-title#8]
I think it would make much sense to have the ability to define a custom output for the handlers.

For instance, if I get `iTALiAN` as language, I'd like it to output `ita` (to normalize it and have always the same string).

The same would be valid for the `source` (with the additions I'm adding).

It would look like:
```
parser.addHandler('language', /\b(ENG(?:LISH)?|eng)\b/, {
  value: 'ita',
});
```

Is it possible?

**Comment by clement-escolano** [comment]
Hello @FezVrasta !

Sorry to for the slow replies to your issues / PRs, I don't have much free time right now.
This is indeed a great idea ! It solves a few problems I had with the package design. I implemented it in the last commit (https://github.com/clement-escolano/parse-torrent-title/commit/13ee2ddd7f2261dd210f28667b6693e27676387e). I also updated the Italian regex.
Does it suits your needs ?

Best
